etc/parameters/default.json

```diff
@@ -465,20 +465,20 @@
         "ankle_roll": 0.05
       }
     },
-    "max_level_delta": 0.1,
-    "max_rotation_speed": 5.0,
+    "max_level_delta": 10000.0,
+    "max_rotation_speed": 50000.0,
```

Comment:
Intended?

Reply:
Yes, this turns it off... This is not the pretty way to do it, but for now sufficient